Userland/Libraries/LibWeb/Layout/BlockContainer.cpp

```diff
@@ -12,30 +13,18 @@ namespace Web::Layout {
 BlockContainer::BlockContainer(DOM::Document& document, DOM::Node* node, NonnullRefPtr<CSS::StyleProperties> style)
     : Box(document, node, move(style))
 {
-    if (dom_node()) {
-        StringBuilder builder;
-        builder.append(dom_node()->class_name());
-        if (MUST(builder.to_string()) == "HTMLInputElement") {
-            // set_natural_width(MUST(CSS::Length(this->dom_node.size()), CSS : Length::Type::Ch).to_px(*this));
-            // set_natural_height(CSS::Length(1, CSS : Length::Type::Lh).to_px(*this));
-            dbgln("We found an input box type of BlockContainer - constructor 1");
-            set_natural_height(400);
-        }
+    if (dom_node() && dom_node()->is_html_input_element()) {
+        set_natural_width(CSS::Length(dynamic_cast<HTML::HTMLInputElement*>(dom_node())->size(), CSS::Length::Type::Ch).to_px(*this));
```

Comment:
Once you've confirmed that the `dom_node()` is indeed an `HTMLInputElement`, you can use `static_cast`, there's no need to use the much slower `dynamic_cast`.